ops/pebble.py

```diff
@@ -333,14 +333,6 @@ class Error(Exception):
     def __repr__(self):
         return '<{}.{} {}>'.format(type(self).__module__, type(self).__name__, self.args)
 
-    def name(self):
-        """Return a string representation of the model plus class."""
-        return '<{}.{}>'.format(type(self).__module__, type(self).__name__)
-
-    def message(self):
-        """Return the message passed as an argument."""
-        return self.args[0]
```

Comment:
Given that this was part of the original contract, I wonder if people might have been doing:

```
try: pebble.Error as e:
  if e.message().contains(...):
```
Certainly it is at least a little dangerous to just remove what would be a public method on a type (which is also why it is dangerous to make arbitrary things public).
It might be better to start this as a deprecation warning?

Reply:
Perhaps, though the intention here was to remove it for the 2.0 release, which may have a (very few) breaking changes. In practice, I think it's rather unlikely people are using this. Quoting my comment at https://github.com/canonical/operator/issues/777:

> I think it's very unlikely that they're being used outside operator. My reasoning: there are only a small number of sidecar charms to begin with, only some of those would catch pebble.Error or its subclasses directly, even fewer of those would call methods on the resulting objects.